editor/import/editor_import_collada.cpp

```diff
@@ -480,7 +480,7 @@ Error ColladaImport::_create_material(const String &p_target) {
 		}
 	}
 
-	float roughness = Math::sqrt(1.0 - ((Math::log(effect.shininess) / Math::log(2.0)) / 8.0)); //not very right..
+	float roughness = -Math::sqrt(1.0 - ((Math::log(effect.shininess) / Math::log(2.0)) / 9.0)) + 1;
```

Comment:
I'd either keep the "not very right" or multiply `effect.shininess` with 512.0/511.0 because that's what it refers to. Also you'd probably want to do

```cpp
float roughness = Math::sqrt(((Math::log(effect.shininess * (512.0/511.0)) / Math::log(2.0)) / 9.0))
```

rather than doing a nonlinear double negation (one under square root and one outside)